### ISSUE TITLE
Create an "empty" state for the messages page

### ISSUE THREAD

**Issue by mewtaylor** [LLK/scratch-www#1499]
Right now, if a user has no messages, the messages page is blank. We should figure out a message to display for users who enter the messages page in this state. 

The state itself will be an edge case, since users receive a notification when they join Scratch. The only scenario in which this could happen is if a users signs in after a year, and has not received any messages in that year, but is viewing the messages page.

**Comment by thisandagain** [comment]
@mewtaylor Good catch. I'm ok to backlog this for now, but this feels like an opportunity to encourage users to be social and engage with other users. 

/cc @carljbowman
